src/composables/transaction.ts

```diff
@@ -142,19 +146,20 @@ export function useTransaction({
 
   const isErrorTransaction = computed(
     (): boolean => {
-      if (!transaction.value) {
+      if (!transaction.value?.tx) {
         return false;
       }
       const { returnType } = transaction.value.tx;
       return !!(returnType && returnType !== RETURN_TYPE_OK);
     },
   );
 
-  const isDex = computed((): boolean => !!(transaction.value
-    && getDexContracts.value && transaction.value.tx.contractId && (
-    getDexContracts.value.router.includes(transaction.value.tx.contractId)
-    || getDexContracts.value.wae.includes(transaction.value.tx.contractId)
-  )));
+  const { wae, router } = getDexContracts.value;
+  const isDex = computed((): boolean => !!(
+    transaction.value?.tx?.contractId
+    && getDexContracts.value
```

Comment:
This test makes no sense as in line 157 the logic assumes that the value of `getDexContracts.value` is an object.